### PR TITLE
2020-03-04 community meeting notes

### PR DESCRIPTION
**Pull request by chris-crone**

### CODE REVIEW

**Review by radu-matei — APPROVED** [review]

**Review by carolynvs — APPROVED** [review]
LGTM

File: meeting-notes/2020-03-04.md [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+    - Schemas currently live online (cnab.io/), we currently fetch the schema to do validation
+    - This doesn't work for airgapped scenarios
+    - Docker has done this in docker/cli for Compose
+    - Radu doesn't want to copy paste the schema in a Go file
+    - Bin packer? gobuffalo/packr
+    - Jacob: Would be good to embed schemas for their parameters
+        - Radu: We could bundle the schema in thick bundles
+            - Matt: Means we'd be asking bundle if it's valid
+- #285 meeting?
+    - Glyn is going to plan this
```
Comment (glyn) [inline meeting-notes/2020-03-04.md:83]: Actually, it's Ralph.

Comment (radu-matei) [reply meeting-notes/2020-03-04.md:83]: cc @squillace

Comment (chris-crone) [reply meeting-notes/2020-03-04.md:83]: I've put up: https://github.com/cnabio/community/pull/27

From the Slack conversation, I thought that it was @glyn that would be organising the meeting. I've updated the minutes and am happy to take setting up the meeting as an action.

Comment (squillace) [reply meeting-notes/2020-03-04.md:83]: it was ralph. And as in channel, we should have it be a formal scheduled meeting item, as we'll chew through it all and leave it in bits

**Pull request merged**